mappings/net/minecraft/nbt/NbtElement.mapping

```diff
@@ -12,6 +8,8 @@ CLASS net/minecraft/class_2520 net/minecraft/nbt/NbtElement
 		COMMENT Gets the type of this NBT element.
 		COMMENT
 		COMMENT @return the type
+		COMMENT
+		COMMENT @see net.fabricmc.yarn.constants.NbtTypeIds for a list of valid types.
```

Comment:
Fyi this renders something like

`for a list of valid types.` as the description for the link to `net.fabricmc.yarn.constants.NbtTypeIds`. You know, javadoc is weird.
```suggestion
		COMMENT @see net.fabricmc.yarn.constants.NbtTypeIds a list of valid types
```